package.json

```diff
@@ -102,7 +102,7 @@
     "ts-node": "^8.4.1",
     "tslint": "^5.20.1",
     "tslint-angular": "^3.0.2",
-    "typescript": "^3.7.4",
+    "typescript": "^3.8.0-dev.20200201",
```

Comment:
Not a huge fan of using dev typescript versions. Currently 3.7.4 isn't even officially supported and is currently stopping us from upgrading the build-angular dependency

Reply:
Reverting to stable is fine